lib/galaxy/datatypes/data.py

```diff
@@ -404,9 +406,8 @@ def display_info(self, dataset):
             if info.find( '\n' ) >= 0:
                 info = info.replace( '\n', '<br/>' )
 
-            # Convert to unicode to display non-ascii characters.
-            if not isinstance(info, unicode):
-                info = unicode( info, 'utf-8')
+            if not isinstance(info, text_type):
+                info = unicodify( info, 'utf-8' )
```

Comment:
> What about just calling:
> 
> info = unicodify( info, 'utf-8' )
> without the if?

I guess so, since `unicodify()` does that anyway. The question is wether `utf-8` is always the correct encoding. There is the theoretical possibility that info is already non-utf-8 unicode ...